src/Psl/Result/Failure.php

```diff
@@ -44,6 +44,11 @@ public function getResult(): void
         throw $this->throwable;
     }
 
+    public function unwrapOr(mixed $default)
```

Comment:
Can you add the (inherited) docblock types here as well?
Also the `: mixed` php return type.


(this should be done on the interface and the 2 implementing classes)